packages/credentials/karma.conf.cjs

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+// Karma is what we're using to run our tests in browser environments
+// Karma does not support .mjs
+
+// playwright acts as a safari executable on windows and mac
+const playwright = require('playwright');
+const esbuildBrowserConfig = require('./build/esbuild-browser-config.cjs');
+
+// use playwright chrome exec path as run target for chromium tests
+process.env.CHROME_BIN = playwright.chromium.executablePath();
+
+// use playwright webkit exec path as run target for safari tests
+process.env.WEBKIT_HEADLESS_BIN = playwright.webkit.executablePath();
+
+// use playwright firefox exec path as run target for firefox tests
+process.env.FIREFOX_BIN = playwright.firefox.executablePath();
+
+module.exports = function (config) {
+  config.set({
+    plugins: [
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
+      'karma-webkit-launcher',
+      'karma-esbuild',
+      'karma-mocha',
+      'karma-mocha-reporter',
+    ],
+
+    // frameworks to use
+    // available frameworks: https://www.npmjs.com/search?q=keywords:karma-adapter
+    frameworks: ['mocha'],
```

Comment:
Increase mocha default timeouts to avoid browser tests failing on GitHub:

```suggestion
    frameworks: ['mocha'],

    // Increase Mocha's default timeout of 2 seconds to prevent timeouts during GitHub CI runs.
    client: {
      mocha: {
        timeout: 10000 // 10 seconds
      }
    },
```